src/lib.rs

```diff
@@ -30,11 +33,16 @@ hypher = { version = "0.1", default-features = false, features = ["english"] }
 ```
 */
 
+#![cfg_attr(not(test), no_std)]
```

Comment:
Why is this necessary? Isn't the only thing we need for testing alloc and you're pulling that in below?

Reply:
I thought it was necessary, but I was wrong